libs/output-mapping/azure-pipelines.tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       dependsOn: [output_mapping_lock]
       displayName: Code Style
       serviceName: ci-output-mapping
-      testCommand: composer phpcs
+      testCommand: composer build
```

Comment:
stejne jako u IM - ten nazev CI stage VS nazev composer stepu VS to co to fakticky dela, nesedi